system_tests_vflow failing in test_03_short_connections

```
64: =========================================================================== FAILURES ===========================================================================
64: ________________________________________________________ VFlowInterRouterTest.test_03_short_connections ________________________________________________________
64: 
64: self = <tests.system_tests_vflow.VFlowInterRouterTest testMethod=test_03_short_connections>
64: 
64:     def test_03_short_connections(self):
64:         """
64:         Generate service traffic from multiple sources (including the router local to the connector)
64:         and verify that BIFLOW records are generated with the expected attributes.
64:         """
64:         success = retry(lambda: self.snooper_thread.match_records({'INTA': [('CONNECTOR', {'START_TIME': ANY_VALUE})]}))
64:         self.assertTrue(success, f"Failed to find baseline connector {self.snooper_thread.get_results()}")
64:     
64:         test_name = 'test_03_short_connections'
64:         flow_count = 100
64:         clients = []
64:         expected = {'EdgeB': []}
64:     
64:         for i in range(flow_count):
64: >           clients.append(EchoClientRunner(test_name, 2, None, None, None, i + 5, 1, port_override=self.tcp_noproc_port_eb, delay_close=False))
64: 
64: /home/runner/work/skupper-router/skupper-router/skupper-router/tests/system_tests_vflow.py:542: 
64: _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
64: /home/runner/work/skupper-router/skupper-router/skupper-router/tests/system_tests_tcp_adaptor.py:159: in __init__
64:     self.e_client = TcpEchoClient(prefix=self.client_prefix,
```

https://github.com/skupperproject/skupper-router/actions/runs/11525836349/job/32188199955#step:34:2882

--- a/tests/system_tests_vflow.py
+++ b/tests/system_tests_vflow.py
@@ -539,19 +539,17 @@ class VFlowInterRouterTest(TestCase):
         expected = {'EdgeB': []}
 
         for i in range(flow_count):
-            clients.append(EchoClientRunner(test_name, 2, None, None, None, i + 5, 1, port_override=self.tcp_noproc_port_eb, delay_close=False))
+            # In this case, the echo clients are not going to send any messages.
+            # The echo clients will simply connect to the router and disconnect immediately.
+            # The router will always accept the connection from the echo clients since there is a
+            # tcpConnector on the other side. The router will terminate the echo client connection
+            # immediately after accepting the connection since there is no echo server on the other side.
+            # The echo clients will terminate (delay_close=False) soon after the router disconnects the connection
+            clients.append(EchoClientRunner(test_name, 2, None, None, None, 0, 0, port_override=self.tcp_noproc_port_eb, delay_close=False))
             expected['EdgeB'].append(('BIFLOW_TPORT', {'END_TIME' : ANY_VALUE, 'SOURCE_HOST' : ANY_VALUE, 'SOURCE_PORT' : ANY_VALUE, 'CONNECTOR' : ANY_VALUE, 'ERROR_CONNECTOR_SIDE' : ANY_VALUE}))
 
         success = retry(lambda: self.snooper_thread.match_records(expected))
         self.assertTrue(success, f"Failed to match records {self.snooper_thread.get_results()}")
-
-        # there is no service present and the connector-side connection attempt
-        # will fail. This will cause the clients to fail since data cannot be
-        # transferred successfully. Ignore these errors:
-
-        for i in range(flow_count):
-            with self.assertRaises(Exception):
-                clients[i].wait()
 
     @classmethod
     def tearDownClass(cls):

